presto-docs/src/main/sphinx/release/release-0.171.rst

```diff
@@ -0,0 +1,28 @@
+=============
+Release 0.171
+=============
+
+General Changes
+---------------
+
+* Fix planning regression for queries that compute a mix of distinct and non-distinct aggregations.
+* Fix casting from certain complex types to ``JSON`` when source type contains ``JSON`` or ``DECIMAL``.
+* Fix issue where queries are not purged after completion, and completion events are not fired
```

Comment:
Should the comma be there? This sounds like two independent fixes. If both only apply to data definition, rephrase like

* Fix issue for data definition queries that prevented firing completion events or purging them from the coordinator's memory.

If separate, write as two separate items.